### PR TITLE
Curator: add reviews pattern

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

Adds Reviews pattern: 

Screen Shot 2022-06-15 at 12 38 33 PM

Shout out to @madhusudhand for the block pattern escaping utility which I used here and it made the process a lot less soul crushing.

#### Related issue(s):

Part of #5935 

Note that the images should probably be compressed but we can do that all at the end.

### CODE REVIEW

File: curator/patterns/reviews.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"10%"} -->
+<div class="wp-block-column" style="flex-basis:10%"></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"center","width":"45%"} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:45%"><!-- wp:paragraph {"fontSize":"x-large"} -->
+<p class="has-x-large-font-size"><?php echo esc_html__( 'The Fisherman', 'curator' ); ?></p>
```
Comment (madhusudhand) [inline curator/patterns/reviews.php]: `h3` spec matching the design and it would be more appropriate to use.

File: curator/patterns/reviews.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+<p><?php echo esc_html__( 'Nearly all the province of Gallura, washed by the Mediterranean on three sides, consists of mountainous tracts, with valleys intervening, similar to this of the Liscia.', 'curator' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size"><strong><?php echo esc_html__( '$$$$', 'curator' ); ?></strong><?php echo esc_html__( ' French, European, Healthy', 'curator' ); ?><br><em><?php echo esc_html__( 'Our review: 5/5', 'curator' ); ?></em></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"style":{"typography":{"textTransform":"uppercase"}}} -->
+<div class="wp-block-button" style="text-transform:uppercase"><a class="wp-block-button__link wp-element-button"><?php echo esc_html__( 'Get Directions →', 'curator' ); ?></a></div>
```
Comment (madhusudhand) [inline curator/patterns/reviews.php]: update fontSize to `medium - 18px`, also fontWeight of the arrow alone seems to be set to `400` in designs.

**Review by madhusudhand — COMMENTED** [review]
I guess we can increase the boundary padding little more to match the space within the pattern.

**Space with other blocks**
image


**Space within**
image

File: curator/patterns/reviews.php (hunk 1 above)
Comment (jffng) [reply curator/patterns/reviews.php]: Good call, updated to h3s.

**Comment by jffng** [comment]
cc @beafialho this is ready for your review.

**Comment by beafialho** [comment]
Thank you, this looks good!

**Pull request merged**